tests/tests/golden-files/test_issue_comment_conventional_commit.yml

```diff
@@ -0,0 +1,14 @@
+input: issue_comment_conventional_commit.json
+output:
+- 'github.IsOrganizationMember: org=mendersoftware,user=tranchitella'
+- 'info:Attempting to amend to satisfy conventional commits in PR: mender/973 and
+  head: dependabot/go_modules/github.com/stretchr/testify-1.7.1'
+- 'git.Run: /usr/bin/git clone -b dependabot/go_modules/github.com/stretchr/testify-1.7.1
+  --single-branch https://github.com/mendersoftware/mender.git'
+- 'git.Run: /usr/bin/git --no-pager show -s --format=%B HEAD'
+- |-
+  git.Run: /usr/bin/git commit --amend -m fix: 
```

Comment:
This does not look quite right to me.

Where is the rest of the commit message?